cb-sol-cli/yarn.lock

```diff
@@ -0,0 +1,116 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Is this in sync with our `package-lock.json`?

Reply:
oh shoot i use yarn , i can remove this. 

pacakge-lock.json directly pulls from package.json anyways.